src/guard.rs

```diff
@@ -3,7 +3,7 @@
 //! # Examples:
 //!
 //! In order to check whether a value would fit in the given
-//! slice without extraneous space:
+//! slice without any extra trailing bytes:
```

Comment:
"with no extra", maybe?

Reply:
Yes, sounds good. Feel free to provide suggestion snippets such as this one:

```suggestion
//! slice with no extra trailing bytes:
```